### PR TITLE
fix(plasma-temple): prefetch slider images

### PR DESCRIPTION
**Pull request by LamaEats**
:baby_chick: Download canary assets:

[colors--canary.228.3514543154.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.228.3514543154.xml)  
[color_sbermarket_ios-swift--canary.228.3514543154.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.228.3514543154.swift)  
[color_sbermarket_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.228.3514543154.kt)  
[color_sbermarket_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.228.3514543154.ts)  
[color_sberprime_ios-swift--canary.228.3514543154.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.228.3514543154.swift)  
[color_sberprime_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.228.3514543154.kt)  
[color_sberprime_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.228.3514543154.ts)  
[PlasmaTokensColor--canary.228.3514543154.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.228.3514543154.swift)  
[typo_mage_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.228.3514543154.kt)  
[typo_mage_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.228.3514543154.ts)  
[typo_plasma_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.228.3514543154.kt)  
[typo_plasma_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.228.3514543154.ts)  
[typo_ruler_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.228.3514543154.kt)  
[typo_ruler_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.228.3514543154.ts)  
[typo_sage_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.228.3514543154.kt)  
[typo_sage_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.228.3514543154.ts)  
[typo_sbermarket_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.228.3514543154.kt)  
[typo_sbermarket_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.228.3514543154.ts)  
[typo_soulmate_kotlin--canary.228.3514543154.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.228.3514543154.kt)  
[typo_soulmate_react-native--canary.228.3514543154.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.228.3514543154.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.112.1-canary.228.3514543154.0
  # or 
  yarn add @salutejs/plasma-temple@1.112.1-canary.228.3514543154.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-228/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-228/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-228/
  ui storybook: http://plasma.sberdevices.ru/pr-228/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-228/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-228/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-228/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-228/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-228/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-228/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-228/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-228/temple-storybook/

  </details>

File: packages/plasma-temple/src/components/HeroSlider/HeroSlider@common.tsx [hunk 1]
```diff
@@ -136,18 +152,26 @@ export const HeroSlider: React.FC<UnifiedComponentProps<HeroSliderProps, Platfor
...
-            setActiveIndex(nextIndex);
-        }, time);
+    useEffect(() => {
+        Promise.all(items.map(({ src }) => fetchMediaSrc(src)));
+    }, [items]);
 
-        return () => {
-            clearTimeout(timerRef.current);
-        };
+    React.useEffect(() => {
```
Comment (Yakutoc) [inline packages/plasma-temple/src/components/HeroSlider/HeroSlider@common.tsx:159]: @LamaEats 

[Minor]

На **155** строчке у тебя используется просто `useEffect ` тогда как здесь  `React.useEffect`. 

Это сделано специально или ? )

**Review by Yakutoc — APPROVED** [review]

**Comment by LamaEats** [comment]
я тут немного подумал и придумал что это не задача пакета, а задача приложения, который использует пакет, так что закрываю :)

**Pull request closed without merging**